package.json

```diff
@@ -7,11 +7,14 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
```

Comment:
Your project is working fine! 
[OPTIONAL] However then, I would suggest you add MIT license to your project before merging it to the main!